### PR TITLE
fix: MD033/no-inline-html

### PR DESCRIPTION
**Pull request by nschonni**
Escape elements swallowed as HTML

### CODE REVIEW

**Comment by nschonni** [comment]
@mairaw rebased

**Comment by nschonni** [comment]
Rebased for the ATX conversion conflicts

**Comment by Rick-Anderson** [comment]
Thanks for the PR. We're not taking any PR's unless they fix an identified problem in the rendered doc.

**Comment by nschonni** [comment]
@Rick-Anderson there are several files in this PR were elements aren't rendering. I get you don't like these changes and won't submit any more PRs here going forward

**Comment by mairaw** [comment]
The changes are good @nschonni. I can see the problem you were fixing it. I'll review them for you!

File: iis/configuration/system.webServer/security/dynamicIpSecurity/denyByRequestRate.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ ms.assetid: 8f0bd47d-adf6-4738-978d-8071b2747048
 msc.legacyurl: /configreference/system.webserver/security/dynamicipsecurity/denybyrequestrate
 msc.type: config
 ---
-# Deny by Request Rate <denyByRequestRate> &lt;denyByRequestRate&gt;
+# Deny by Request Rate &lt;denyByRequestRate&gt;
```
Comment (mairaw) [inline iis/configuration/system.webServer/security/dynamicIpSecurity/denyByRequestRate.md:10]: you should also remove this from the title metadata here no?

**Pull request merged**